index.html

```diff
@@ -14,6 +14,16 @@ <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
+      <nav id="mobile-nav">
+        <button type="button" class="menu-bars">
+          <i class="fas fa-bars"></i>
+        </button>
+        <ul>
+          <li><a class="nav-link mobile-links" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link mobile-links" href="#about-me">About</a></li>
+          <li><a class="nav-link mobile-links" href="#contact">Contact</a></li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Well done on the mobile menu 👏🏽 , yet as shown below, your design is slightly different than the provided Figma template, the project's [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) 🔽 state that you should always stick to the Figma template as much as possible, please try to work on it.
> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### Figma Design
![Figma design](https://user-images.githubusercontent.com/104319462/218040077-10e7fd3d-5601-4522-a258-6c80e31a57f8.png)

### Your Design
![PR design](https://user-images.githubusercontent.com/104319462/218041603-5c160b7d-33a6-4fb1-a33c-b9e911928217.png)